components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/CertificateValidationUtil.java

```diff
@@ -885,7 +886,8 @@ public static RevocationStatus getRevocationStatus(X509Certificate peerCert, X50
             for (AccessDescription accessDescription : accessDescriptions) {
 
                 GeneralName gn = accessDescription.getAccessLocation();
-                if (gn.getTagNo() == GeneralName.uniformResourceIdentifier) {
+                if (gn.getTagNo() == GeneralName.uniformResourceIdentifier &&
+                        accessDescription.getAccessMethod().equals(X509ObjectIdentifiers.ocspAccessMethod)) {
```

Comment:
is `X509ObjectIdentifiers.ocspAccessMethod` a constant? If that's a const, can we change the order?

Reply:
Addressed in https://github.com/wso2-extensions/identity-x509-commons/pull/13/commits/378a1efb68e4dabc81182abef36294921ec9fd32